addons/website_slides/controllers/mail_thread.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class MailThread(models.AbstractModel):
+    _inherit = 'mail.thread'
+
+    @api.multi
+    @api.returns('mail.message', lambda value: value.id)
+    def message_post(self, **kwargs):
```

Comment:
Do we want this applied also to users posting a message in the chatter ? 
If no, I suggest we move this in the controller (see /mail/chatter_post root to extend then).
cc @tde-banana-odoo , as it implies message_post.